platform/src/main/java/org/hillview/utils/TestTables.java

```diff
@@ -18,6 +18,7 @@
 
 package org.hillview.utils;
 
+import com.sun.tools.javac.util.Assert;
```

Comment:
You should use the unit test assert; please look at the other test files to see which import they use.